### PR TITLE
#1848 - Documentation improvements

### PR DESCRIPTION
**Pull request by jcklie**
**What's in the PR**
* Add more documentation for Intellij

**How to test manually**
* Build and then read the documentation

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [X] PR updates documentation

### CODE REVIEW

File: inception-doc/src/main/resources/META-INF/asciidoc/developer-guide/setup.adoc [hunk 1]
```diff
@@ -225,6 +225,13 @@ way is to go to `File -> Open` and the select the `pom.xml` in the {product-name
 IntelliJ IDEA will then guide you through the import process, the defaults work out of the box.
 INCEpTION can now be started via running `inception-app-webapp/src/main/java/de/tudarmstadt/ukp/inception/INCEpTION.java`.
 
+If you get errors that certain classes are not found, then open a terminal, go to the INCEpTION
+repository root and run
+
+    mvn clean install -DskipTests=true -D "checkstyle.skip=true"
```
Comment (reckart) [inline inception-doc/src/main/resources/META-INF/asciidoc/developer-guide/setup.adoc:231]: Shouldn't that be `-Dcheckstyle.skip=true`? Why the quotes and the space after `-D`?

File: inception-doc/src/main/resources/META-INF/asciidoc/developer-guide/setup.adoc [hunk 2]
```diff
@@ -225,6 +225,13 @@ way is to go to `File -> Open` and the select the `pom.xml` in the {product-name
...
+
+Alternatively, you can run the `install` Maven goal from IntelliJ.
```
Comment (reckart) [inline inception-doc/src/main/resources/META-INF/asciidoc/developer-guide/setup.adoc:233]: No `clean` here?

File: inception-doc/src/main/resources/META-INF/asciidoc/developer-guide/setup.adoc (hunk 1 above)
Comment (jcklie) [reply inception-doc/src/main/resources/META-INF/asciidoc/developer-guide/setup.adoc]: I changed it,both works

File: inception-doc/src/main/resources/META-INF/asciidoc/developer-guide/setup.adoc (hunk 2 above)
Comment (jcklie) [reply inception-doc/src/main/resources/META-INF/asciidoc/developer-guide/setup.adoc]: I added clean

**Review by reckart — APPROVED** [review]

**Pull request merged**